Cargo.toml

```diff
@@ -43,7 +43,6 @@ hex = "0.4"
 kdbush = "0.2"
 log = "0.4.19"
 lyon = "1.0.1"
-polylabel = "2.5"
```

Comment:
I'm guessing you've removed this example because it uses FGB. We could convert it to use geojson, but I'm guessing you've judged it to be not worth the upkeep?

That seems fine, just making sure I understand.

Reply:
Exactly. This example should go either into the flatgeobuf or the polylabel crate. I tried to use geojson, but I had  the same problem with the missing feature iterator.